packages/core/src/Tab/index.tsx

```diff
@@ -154,12 +154,27 @@ export function TabBase<T>({
     onSelect(id)
   }, [selected, disabled, id, onSelect])
 
+  const handleKeyDown = useCallback<React.KeyboardEventHandler<BaseElement>>(
+    event => {
+      if (selected || disabled) {
```

Comment:
It should be still possible to pass `onKeyDown` to the component so you should have above this line `      onKeyDown?.(event)`

Reply:
Ok, added the line.

Reply:
👍